SignUp/SignUp/ThirdViewController.swift

```diff
@@ -28,17 +33,40 @@ class ThirdViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        pnumField.text = UserInformation.shared.pnum
+        dateLabel.text = UserInformation.shared.birth
+        self.joinBtn.isEnabled = false
+        self.pnumField.delegate = self
         // Do any additional setup after loading the view.
     }
     
 
+    
+    func textFieldDidEndEditing(_ textField: UITextField) {
+        if ( pnumField.text?.isEmpty == false ) && ( dateLabel.text?.isEmpty == false ) == true {
+            self.joinBtn.isEnabled = true
+        } else {
+            self.joinBtn.isEnabled = false
+        }
```

Comment:
```suggestion
        self.joinBtn.isEnabled = ( pnumField.text?.isEmpty == false ) && ( dateLabel.text?.isEmpty == false )
```
꼭 이렇게 해야한다는것은 아니고 이런 방법도 있다~ 라고 생각해주시면 감사하겠습니다